components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1181,6 +1184,35 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
         return dataDO;
     }
 
+    private void getAccessTokenExtendedAttributes(String accessTokenIdentifier, AccessTokenDO dataDO) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Retrieving extended attributes for access token: " + accessTokenIdentifier);
+        }
+        if (dataDO == null) {
+            return;
```

Comment:
add some debug logs here. Can this DO be null?

Reply:
Removed this as this cannot be null
ce641ed230aab6181e59cf0439df3e52b17290bb